__tests__/reducers/fetchReducer.test.js

```diff
@@ -0,0 +1,34 @@
+import fetchReducer from "../../front-end/src/reducers/fetchReducer";
+let initialState;
+beforeEach(() => {
+  initialState = {
+    isFetching: false,
+    error: ''
+  }
+});
+
+test('should set default state', () => {
+    const state = fetchReducer(undefined, { type: '@@INIT' });
+    expect(state).toEqual(initialState);
+});
+
+test('should set team members', () => {
```

Comment:
Throughout the tests, each test should have a unique name, not just in this file.